developer-docs-site/docs/indexer/txn-stream/local-development.md

```diff
@@ -108,3 +114,6 @@ Try setting the following environment variable before running the script:
 ```bash
 export REDIS_IMAGE_REPO=arm64v8/redis
 ```
+
+### Cache worker is crashlooping or `Redis latest version update failed.` in log
+Wipe the Data.
```

Comment:
Can you include a command to show how to do this?

Reply:
done.